scripts/prepare_doc_publication.py

```diff
@@ -54,6 +54,7 @@ def parse_arguments():
     'getting_started_windows_bazel.md': 'Windows with Bazel',
     'getting_started_windows_cmake.md': 'Windows with CMake',
     'getting_started_windows_vulkan.md': 'Windows with Vulkan',
+    'developer_overview.md': 'Developer Overview',
```

Comment:
No need to explicitly list this if it matches the H1 of the MarkDown. The default behavior is to grab the H1 as the title. :)